packages/shared/chat/sagas.ts

```diff
@@ -289,9 +289,14 @@ function initChatCommands() {
     }
   })
 
-  addChatCommand('showfps', 'Show FPS counter', (_message) => {
+  addChatCommand('debug', 'Show debug panel', (_message) => {
```

Comment:
Can we keep both? People have muscular memory for showfps

Reply:
I don't know if it's a good idea to have both because it means 2 functions that do the same thing, so there is more "noise" in the chat /help command. Debug in my opinion makes more sense because it's a bit more descriptive, but if people are used to the /showfps we can keep it like that. The change was requested by @olavra let's see what he thinks as well

Reply:
Sorry this is a no-go, we should keep the previous one as well

Reply:
Alright :) in this case if I can, I'd keep just the old one and avoid having the /debug one to avoid confusion